platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -393,7 +395,8 @@ private VirtualRootNode(VirtualRootNode<K, V> source) {
     @SuppressWarnings("ClassEscapesDefinedScope")
     public void postInit(final VirtualStateAccessor state) {
         // We're reconnecting, state doesn't match cache or dataSource, gotta bail.
-        if (learnerTreeView != null) {
+        //        if (learnerTreeView != null) {
+        if (originalMap != null) {
             fullyReconnectedState = state;
```

Comment:
Please remove commented-out code. Also, it's not fully clear as to why these two conditions are considered equivalent. Could you please elaborate?

Reply:
Yes, I'll remove it.

This change comes from what I mentioned in PR description: the virtual root node now creates its learner tree view in `buildLearnerTreeView()` rather than in `setupWithOriginalNode()`. In between these calls, postInit() is called. Previously, when `learnerTreeView` was initialized in `setupWithOriginalNode()`, this condition in `postInit()` was true, now `learnerTreeView` is still null here. This is why I had to change it to `originalMap`, which is still created in `setupWithOriginalNode()`

Reply:
Got it. Makes sense. Thanks for the clarification.